README.md

```diff
@@ -5,17 +5,18 @@ temporalts is a module that leverages [Moment](https://www.npmjs.com/package/mom
 ## Installation
 
 ```
-npm install timelts
+npm install temporalts
```

Comment:
i think this was left from an earlier version? i didn't see timelts anywhere else in the module and it's `temporalts` in the `package.json` 🤔

Reply:
indeed it was. Thank you.